### ISSUE TITLE
Allow formatting advanced legend total value

### ISSUE THREAD

**Issue by mzrimsek** [swimlane/ngx-charts#794]
**I'm submitting a ...**  (check one with "x")
```
[ ] bug report => search github for a similar issue or PR before submitting
[x] feature request
[ ] support request => Please do not submit support request here
```

**Current behavior**
Currently there is an option to set what the total label will be, but not ability to format the numerical value like you can with each individual legend item

**Expected behavior**
Should be able to pass in a formatting function similar to the value formatting option to format the numerical value in the total section of the legend

**What is the motivation / use case for changing the behavior?**
This would provide more flexibility in formatting data output. My specific use case is that I have time in seconds that I am able to format into a readable format for the legend items, but not for the total.

* **ngx-charts version:** 7.2.0

* **Angular version:** 5.2.0

* **Language:** TypeScript 2.5.3

**Comment by mzrimsek** [comment]
Potentially this could be remedied by just reusing the valueFormatting function already being passed in.

**Comment by tdmitchal** [comment]
Can you give an example???

**Comment by mzrimsek** [comment]
Take this data set:
```json
[{
	"name": "Point 1",
	"value": 60
}, {
	"name": "Point 2",
	"value": 120
}, {
	"name": "Point 3",
	"value": 180
}]
```

If you pass this into the advanced pie chart graph it will display 3 separate pie charts for each point, as well as some stuff about the total.

In my use case my values are seconds, so I was easily able to utilize the valueFormatting option to pass in a function to format my seconds into something like "00:01:00" and so on. This was not the case with the number displayed for the total.

Through looking at the code, it appears that this is just the aggregate value from all of the data points - the same variable that is used to calculate the percentage of each individual pie chart. My proposal is that there be an option introduced to format the total value as well. I have looked into it a little bit, but am unsure as to exactly the best way to approach adding this functionality.

In an ideal world the current total variable would remain unchanged, but this new input option would just format the value in place where it is being rendered.

Does that help? If not I can write up a small example to showcase what I am talking about more specifically.

**Comment by marjan-georgiev** [comment]
Merged #906 
This will be in the next relase.
